test/end-to-end/pages/screeners/applicantexecutor.js

```diff
@@ -1,19 +1,23 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
-const content = require('app/resources/en/translation/screeners/applicantexecutor');
+const contentEn = require('app/resources/en/translation/common');
+const contentCy = require('app/resources/cy/translation/common');
+const applicantExecutorEn = require('app/resources/en/translation/screeners/applicantexecutor');
+const applicantExecutorCy = require('app/resources/cy/translation/screeners/applicantexecutor');
 
-module.exports = async function(answer) {
+module.exports = async function(language ='en', answer) {
     const I = this;
+    const commonContent = language === 'en' ? contentEn : contentCy;
+    const applicantExecutorContent = language === 'en' ? applicantExecutorEn : applicantExecutorCy;
 
     await I.checkPageUrl('app/steps/ui/screeners/applicantexecutor');
-    await I.waitForText(content.question);
-    await I.see(content.hintText1);
-    await I.see(content.hintText2);
-
+    if (language === 'en') {
+        await I.waitForText(applicantExecutorContent.question);
+        await I.see(applicantExecutorContent.hintText1);
+        await I.see(applicantExecutorContent.hintText2);
+    }
```

Comment:
again, not sure why you'd only want to check the text here on English journeys? It will likely help with test stability for both languages

Reply:
don't think we need if statement